### DISCUSSION TITLE
0.61.5

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#8565] [General]
* Fix potential `wslservice.exe` crash when tearing down VM state.

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/0.61.5'>0.61.5</a>.</em>

**Comment by win98se** [comment]
Hi! I have encountered a bug with WSLg. More details at https://github.com/microsoft/wslg/issues/782. Thank you in advance!
